src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Badging for App Icons
+subhead: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen.
+authors:
+  - petelepage
+description: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen. Badging makes it easy to subtly notify the user that there is some new activity that might require their attention, or it can be used to indicate a small amount of information, such as an unread count.
+date: 2018-12-11
+updated: 2019-09-26
+tags:
+  - post
+  - capabilities
+  - badging
+  - progressive-web-apps
+  - notifications
+  - origintrials
+  - fugu
+hero: hero.jpg
+alt: Phone showing several notification badges
+---
+
+{% Aside 'caution' %}
+  We're currently working on this API as part of the new
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  and starting in Chrome 73 it is available as an [**origin trial**](#ot).
+  This post will be updated as the Badging API evolves.
+{% endAside %}
+
+## What is the Badging API? {: #what }
+
+<figure class="w-figure w-figure--inline-right" style="max-width:50%">
+  <img  src="badges-on-windows.jpg">
+  <figcaption>
+    Example of Twitter with eight notifications and another app showing a flag
+    type badge.
+  </figcaption>
+</figure>
```

Comment:
One option would be to move the image underneath the first paragraph of text and center it so it can be larger?

Reply:
I'd prefer to keep it half-width if possible. Any objections?

Reply:
I agree. We should have a way to do this so we can control white space on larger screens.

Reply:
The image will attempt to center itself on mobile, but since the image is 900px it's taking up the full width. This is fine on a phone but looks a bit weird on a tablet. I think if you crop the image to something like 600-400px it would work fine for the floating case, and also center nicely on mobile and tablets.

Reply:
Resized to 600px